### ISSUE TITLE
Auto-Expand all submitted files in the TA grading view

### ISSUE THREAD

**Issue by bmcutler** [Submitty/Submitty#871]
Either a setting configured by the instructor, or a single button press by the grader.

Will be especially helpful for grading short answer quizzes.

**Comment by maksimt** [comment]
Meanwhile... for those currently using the system who don't want to wait, install Greasemonkey/Tampermonkey and the following script (currently configured to only open PDF):

```// ==UserScript==
// @name         Autoexpand Submitty Submissions
// @namespace    http://tampermonkey.net/
// @version      0.1
// @match        https://hwgrading.cs.rpi.edu/account/index.php?g_id*
// @include      https://hwgrading.cs.rpi.edu/account/index.php?g_id*
// @require         http://ajax.googleapis.com/ajax/libs/jquery/1.6.2/jquery.min.js
// ==/UserScript==

//--- Note that the contains() text is case-sensitive.
clickLinkWithText('submissions');
clickLinkWithText('pdf');

function clickLinkWithText (linkText) {
    var targetLink = $("a:contains('" + linkText + "')");
    if (targetLink.length) {
        triggerMouseEvent (targetLink[0], "click");
    }
}

function triggerMouseEvent (node, eventType) {
    var clickEvent = document.createEvent('MouseEvents');
    clickEvent.initEvent (eventType, true, true);
    node.dispatchEvent (clickEvent);
}

**Comment by bmcutler** [comment]
Closed in PR #942
